src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+    private final Color color;
+
+    public RightTriangle(Color color, double firstLeg, double secondLeg) {
+        this.color = color;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rightTriangle, area: "
+                + calculateArea() + "sq. units, firstLeg: "
+                + firstLeg + " units, " + "secondLeg: "
+                + secondLeg + " units, "
+                + "color: " + color.name());
+    }
+}
```

Comment:
Don't begin class or method implementation with an empty line. There are redundant empty lines at the beginning of the class and methods.